llvm/include/llvm/CodeGen/GlobalISel/LegalizationArtifactCombiner.h

```diff
@@ -906,7 +906,7 @@ class LegalizationArtifactCombiner {
                                         unsigned &DefOperandIdx) {
       if (Register Def = findValueFromDefImpl(Reg, 0, Size)) {
         if (auto *Unmerge = dyn_cast<GUnmerge>(MRI.getVRegDef(Def))) {
-          DefOperandIdx = Unmerge->findRegisterDefOperandIdx(Def);
+          DefOperandIdx = Unmerge->findRegisterDefOperandIdx(Def, nullptr);
```

Comment:
Please can you use /*TRI=*/nullptr to make it?

Reply:
fixed